client/my-sites/patterns/hooks/use-patterns.ts

```diff
@@ -8,7 +8,7 @@ export function getPatternsQueryOptions(
 	queryOptions: Omit< UseQueryOptions< Pattern[] >, 'queryKey' > = {}
 ) {
 	return {
-		queryKey: [ 'patterns', 'library', locale, category ],
+		queryKey: [ 'pattern-library', 'patterns', locale, category ],
```

Comment:
We could update this [query key](https://github.com/Automattic/wp-calypso/blob/3ca9bedc958dd1ac063667aed31e07fbea3382e2/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-assembler-patterns.ts#L12) as well to follow the same approach:

```javascript
queryKey: [ 'pattern-assembler', lang ],
```